pkg/config/validation/validation.go

```diff
@@ -1165,7 +1165,8 @@ func validatePolicyTargetReference(targetRef *type_beta.PolicyTargetReference) (
 		v = appendErrorf(v, "targetRef name must be set")
 	}
 	if targetRef.Namespace != "" {
-		v = appendErrorf(v, "targetRef namespace must not be set")
+		// cross namespace referencing is currently not supported
+		v = appendErrorf(v, "targetRef namespace must not be set; local namespace is inferred from the policy namespace")
```

Comment:
is it ok to just mention that cross namespace is not supported? This seems helpful for the user to know so they don't try other things like setting the name to `namespace/name`.

```suggestion
		v = appendErrorf(v, "targetRef namespace must not be set; cross namespace attachment is not supported")
```

Reply:
I considered that; but then decided not to because I was worried about leaking too much info not relevant to the user?

But when you put it that way, I'd prefer for them to know if it means not doing `namespace: namespace/name`.

I'll do this change with a slight change for consistency. Good call!